_data/other.yml

```diff
@@ -606,11 +606,12 @@ websites:
       - totp
     doc: https://www.sanebox.com/help/127
 
-  - name: SignNow
+  - name: signNow
     url: https://www.signnow.com/
-    img: signnow.png
-    twitter: signnow
-    facebook: esignaturesignnow
+    img: signnow.svg
+    tfa:
+      - email
+    doc: https://www.signnow.com/features/setup-dual-factor-authentication
```

Comment:
```suggestion

```
I think we can remove the doc link - it doesn't have any useful info about the 2FA features that you showed in the screenshot.